.github/workflows/deploy.yml

```diff
@@ -0,0 +1,60 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - production
+      - develop
+
+env:
+  DEPLOY_DIR: ${{ github.ref == 'refs/heads/production' && '/var/www/link-sharing' || '/var/www/link-sharing-develop' }}
+  SERVER_IP: ${{ secrets.SERVER_IP }}
+  SERVER_USER: ${{ secrets.SERVER_USER }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up SSH key
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan ${{ env.SERVER_IP }} >> ~/.ssh/known_hosts
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
```

Comment:
not PHP 8.3? The other workflow uses PHP 8.3

Reply:
Good eye!